tidb-cloud/branch-github-integration.md

```diff
@@ -1,71 +1,73 @@
 ---
-title: Branching GitHub Integration
-summary: Learn how to use the Branching GitHub integration.
+title: Integrate TiDB Serverless Branching with GitHub
```

Comment:
Integrate TiDB Serverless Branching or TiDB Serverless Branch with GitHub?

I prefer TiDB Serverless Branch, branching refers to the whole integration workflow.

Reply:
If we use "Integrate TiDB Serverless Branching with GitHub", it gives the impression that we need to first create a branch for TiDB Serverless, and then integrate the branch with GitHub. But the integration process does not require users to create a branch in advance.

Reply:
<img width="786" alt="image" src="https://github.com/pingcap/docs/assets/52435083/03f1042f-97b6-48e9-9eb7-a551c9829ffe">
<img width="770" alt="image" src="https://github.com/pingcap/docs/assets/52435083/4673c5cf-9316-43d1-afc7-26a96a36cc11">

Reply:
branching means a feature or the whole integration workflow. I prefer branching here

Reply:
+1